tasks/components.py

```diff
@@ -13,8 +13,8 @@
 
 from tasks.libs.types.copyright import COPYRIGHT_HEADER
 
-Component = namedtuple('Component', ['path', 'doc', 'team'])
-Bundle = namedtuple('Bundle', ['path', 'doc', 'team', 'components'])
+Component = namedtuple('Component', ['name', 'def_file', 'path', 'doc', 'team', 'version'])
+Bundle = namedtuple('Bundle', ['path', 'doc', 'team', 'content', 'components'])
```

Comment:
Should we add a comment explaining what version 1, and 2 means?

Reply:
Sure, added an explanation